resources/qml/Preferences/MaterialView.qml

```diff
@@ -108,15 +108,15 @@ TabView
                 {
                     width: scrollView.columnWidth;
                     height:  parent.rowHeight;
-                    spacing: UM.Theme.getSize("default_margin").width/2
+                    spacing: parseInt(UM.Theme.getSize("default_margin").width/2)
```

Comment:
`Math.floor()`

Reply:
I agree that Math.floor() is better. I think parseInt will first cast the value to a string, and then parse it back to a number again. I used | 0 as shorthand for Math.floor() elsewhere, but that's a little obscure.

Reply:
We found the | 0's indeed, but they're easily mistaken for || 0 (yay JS syntax). We can use Math.floor indeed.

Reply:
I ll replace all parseInt() with Math.floor()